velox/dwio/parquet/tests/reader/ParquetReaderBenchmark.cpp

```diff
@@ -136,6 +152,11 @@ class ParquetReaderBenchmark {
     return rowReader;
   }
 
+  // This method is the place where we do the read opeartion.
+  // scanSpec contains the setting of filter and null-filter.
+  // e.g. filter 30% filter 70% of data and 30% remain.
+  // e.g. null-filter 70% filter 70% of data and 30% remain.
```

Comment:
What do you mean null-filter? Why does it have opposite meaning with the filterRate? This is very confusing.

Reply:
I just want to explain what filterRateX100 and nullsRateX100 are. It is hard to a new person to understand these rates. The nullsRateX100 has opposite meaning with the filterRateX100.